css-animations-1/animation-iteration-count-002.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Animations Test: animation-iteration-count - infinite value</title>
+<link rel="author" title="Nokia Inc." href="http://www.nokia.com">
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="http://dev.w3.org/csswg/css-animations-1/#animation-name">
+<link rel="help" href="http://dev.w3.org/csswg/css-animations-1/#animation-duration">
+<link rel="help" href="http://dev.w3.org/csswg/css-animations-1/#animation-iteration-count">
+<meta name="flags" content="animated">
+<meta name="assert" content="When animation-iteration-count is set to infinite,
+                             animation will repeat forever.">
+<style>
+  div {
+    animation-name: sample;
+    animation-duration: 10s;
+    animation-iteration-count: infinite;
+
+    background-color: blue;
+    height: 100px;
+    width: 100px;
+    position: relative;
+  }
+
+  @keyframes sample {
+    from {
+      left: 150px;
+    }
+    to {
+      left: 0px;
+    }
+  }
+</style>
+<body>
+  <p>
+    Test passes if there is a filled square with 'Filler Text',
+    which starts moving from right to left, then back to right and moves from
+    right to left again. This cycle gets repeated.
```

Comment:
Hmm, 'cycle' here means 2 iteration counts. I prefer the existing pass condition description.